src/routes/safe/components/Transactions/helpers/TxParametersDetail/index.tsx

```diff
@@ -36,6 +51,11 @@ const StyledButtonLink = styled(ButtonLink)`
   }
 `
 
+const StyledDivider = styled(Divider)`
+  width: calc(100% + 32px);
```

Comment:
Alternatively you can also set `margin-right: -16px` to "stretch" the element without setting the width